src/modules/simulator/gyrosim/gyrosim.cpp

```diff
@@ -109,6 +109,20 @@ using namespace DriverFramework;
 #define EXTERNAL_BUS 0
 #endif
 
+namespace
+{
+/** Save way to check if float is zero */
+inline bool isZero(const float &val)
+{
+	return abs(val - 0.0f) < FLT_EPSILON;
```

Comment:
fabsf?

Reply:
abs works also for floating point number with a C++ sdlib. But we don’t have it on Nuttx (IIRC)? I’ll make it C compliant.